Getting Started Docs: Add `Upgrade` page (#77717)
vercel/next.js: docs/01-app/01-getting-started/12-upgrading.mdx

---
title: Upgrade
description: Learn how to upgrade your Next.js application to the latest version.
related:
  title: Version guides
  description: See the version guides for in-depth upgrade instructions.
  links:
    - app/building-your-application/upgrading/canary
    - app/building-your-application/upgrading/version-15
    - app/building-your-application/upgrading/version-14
---

## Latest version

To update to the latest version of Next.js, you can use the `upgrade` codemod:

```bash filename="Terminal"
npx @next/codemod@canary upgrade latest
```

> If you prefer to do it manually, install the latest Next.js and React versions:

Reviewer: ```suggestion
If you prefer to upgrade manually, install the latest Next.js and React versions:
```
